docs/api/turn-by-turn/api-reference.md

```diff
@@ -106,6 +106,7 @@ These options are available for `auto`, `auto_shorter`, `bus`, and `truck` costi
 | `country_crossing_cost` | A cost applied when encountering an international border. This cost is added to the estimated and elapsed times. The default cost is 600 seconds. |
 | `country_crossing_penalty` | A penalty applied for a country crossing. This penalty can be used to create paths that avoid spanning country boundaries. The default penalty is 0. |
 | `shortest` | Changes the metric to quasi-shortest, i.e. purely distance-based costing. Note, this will disable all other costings & penalties. Also note, `shortest` will not disable hierarchy pruning, leading to potentially sub-optimal routes for some costing models. The default is `false`. |
+| `ignore_closures` | If set to `true`, ignores all closures, marked due to live traffic closures, during routing. **Note:** This option cannot be set if `location.search_filter.exclude_closures` is also specified |
```

Comment:
The note here should explain also what happens if both options are specified (looking at the code in worker.cc it seems an exception is thrown).

Reply:
The exception will get translated to an internal error code (`143` in this case), and that further translates to a [HTTP status 400 later](https://github.com/valhalla/valhalla/pull/2685/files#diff-dda1eacb6be34d7946468f09844f48890c1266509f634990ec27263271bd6f48R99).

Looking at the docs, it seems that all error codes are explained later in a separate section rather inline with options that can possibly create those errors.